src/test/java/com/serenitydojo/fruitmarket/ShoppingCart.java

```diff
@@ -0,0 +1,64 @@
+package com.serenitydojo.fruitmarket;
+
+import com.serenitydojo.*;
+import org.junit.Before;
+import org.junit.Test;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+public class ShoppingCart {
+    Catalog catalog = new Catalog();
+    @Before
+    //setup catalog
+    public void setCatalog(){
+        catalog.setPriceOf(Fruit.APPLE,6.50);
+        catalog.setPriceOf(Fruit.ORANGE, 5.50);
+        catalog.setPriceOf(Fruit.BANANA, 6.00);
+        catalog.setPriceOf(Fruit.PEAR, 4.50);
+    }
+
+
+    @Test
+    public void shouldCartbeEmpty() {
+        Cart cart = new Cart(catalog);
+
+        int numberOfItems = cart.getNumberofItems();
+        assertThat(numberOfItems).isEqualTo(0);
+    }
+
+    @Test
+    public void shouldCartHaveItems(){
+        Cart cart = new Cart(catalog);
+        cart.addtoCart(new CartItem(Fruit.APPLE,1));
+        cart.addtoCart(new CartItem(Fruit.BANANA,2.5));
+        cart.addtoCart(new CartItem(Fruit.PEAR,6));
+
+        double numberofUnits = cart.getTotalUnits();
```

Comment:
i might name that `cart.getTotalQuantity()` because you are calling `item.getQuantity()`